### DISCUSSION TITLE
S3 Support using BIOS hack

### DISCUSSION THREAD

**Discussion by Mateusz-Gwara** [jrandiny/yoga-slim7-ubuntu#39] [Q&A]
Hi,

I've enabled S3 Support via the BIOS unlock method.
Ubuntu seems to perform suspend-mode prpperly after this.

Do I need to perform step 6. "Set the default sleep type to S3 (deep)" from the suspend workaround tutorial or does the bios setting suffice?

I'm using Ubuntu 20.04.2 with 5.8.x kernel

**Comment by jrandiny** [comment]
Sorry for the late reply, after using the bios unlock method, Ubuntu should defaulted to deep sleep. To confirm, you can run `cat  /sys/power/mem_sleep`
